build.zig

```diff
@@ -44,13 +44,18 @@ pub fn build(b: *Builder) !void {
         .target = target,
         .optimize = optimize,
     }).module("xev");
+    const datetime = b.dependency("datetime", .{
+        .target = target,
+        .optimize = optimize,
+    });
 
-    // add zeam-params
+    // add zeam-utils
     const zeam_utils = b.addModule("@zeam/utils", .{
         .target = target,
         .optimize = optimize,
         .root_source_file = b.path("pkgs/utils/src/lib.zig"),
     });
+    zeam_utils.addImport("datetime", datetime.module("datetime"));
```

Comment:
instead of doing .module later on at 2 places, just .module when you are first creating datetime